gef.py

```diff
@@ -1001,9 +1087,9 @@ def get_libc_version():
     return 0, 0
 
 
-def get_main_arena():
+def get_glibc_arena(addr=None):
```

Comment:
Could you profile this? I think that we could cache/memoize it if it's slow.

Reply:
I just did a little benchmark with 10.000 iterations each:

``´
[not cached] 0.6421656608581543 s
[cached] 0.0005979537963867188 s
```
For e.g. the `heap chunks` command this is only executed once per arena this doesn't seem like a big bottleneck but we can cache it as well :)

Reply:
If I want to run it a bunch of times I don't want to wait half a second each time. Let's cache it :)

Reply:
oh, the times are for 10.000 times each. so if you run it around 10.000 times its around half a second. but I already pushed a cached version :)

Reply:
Oh, 0.6 seconds for 10'000 iterations :) So it's not too big of a deal.

If someone is poking around and modifying the heap it should hit the `memchanged_handler` which clears the cache so there should be only benefit to keeping it cached UNLESS the arena object itself is modified. I don't think we do that, is that correct?

Reply:
yes, after parsing we never modify the arenas. so caching should not be a problem.